slow_odgi/mygfa.py

```diff
@@ -91,19 +91,41 @@ def __str__(self):
             str(self.overlap),
         ])
 
+@dataclass
+class SegO:
```

Comment:
I realize this looks like a sprawling commit. It's all basically just the claeanup after getting this little dataclass into mygfa.

Reply:
`SegO` is a fine name, but I think a "segment/orientation pair" is what we also know as a "handle"? (At least according to [the relational doc](https://github.com/cucapra/notes/blob/main/Pollen/backend/relational.md#other-types).) It's a confusing name, obviously, but it is what folks seem to call it.

Reply:
🙈 Wow a little embarrassing! I'll change it!